UsageDoc/CA_UsageDocument.md

```diff
@@ -7,10 +7,9 @@
 
 * [Clearing Automation tool workflow diagram](#clearing-automation-tool-workflow-diagram)
 
-
 * [Prerequisite](#prerequisite)
 
-* [Package Installation](#package-installation) 
+* [Package Installation](#installation) 
 
 * [Demo Project](#demo-project-after-consuming-the-package)
 
```

Comment:
For Clearing Automation workflow hyperlinks can be given to container execution steps and package execution step instead of splitting as identifier,- creator and uploader